### PR TITLE
Automate documentation generation

### PR DESCRIPTION
**Pull request by justetz**

### CODE REVIEW

**Comment by ddbruce** [comment]
Please modify the Dockerfile so it runs properly. I believe the line in question is https://github.com/rpiambulance/at-channel/blob/8de2bdfbc08923d9a34bb0377b83533614de1fa5/Dockerfile#L6

File: Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ RUN mkdir -p /usr/src/app
 
 WORKDIR /usr/src/app
 
-COPY . /usr/src/app/
+COPY ./src/ /usr/src/app/
```
Comment (ddbruce) [inline Dockerfile:7]: Thanks!

**Comment by ddbruce** [comment]
@zapdos26 `npm start` defaults to `node server. js`

**Comment by zapdos26** [comment]
> @zapdos26 `npm start` defaults to `node server. js`

It does not work, since server.js is inside /src/. I tested it by both building the container and running it locally

**Comment by ddbruce** [comment]
> It does not work, since server.js is inside /src/. I tested it by both building the container and running it locally

Ahhhh yes yes, that makes very much sense.

@justetz I rescind my approval and request the appropriate changes 😄

**Comment by zapdos26** [comment]
Another error:
`npm ERR! enoent ENOENT: no such file or directory, open '/usr/src/app/package.json'`

File: Dockerfile (hunk 1 above)
Comment (zapdos26) [inline Dockerfile:7]: This should just be a `.`

**Pull request merged**